RUFAS/routines/animal/genetics/animal_genetics.py

```diff
@@ -1,15 +1,248 @@
+
+        Parameters
+        ----------
+        original_net_merit: dict[str, dict[str, dict[str, float]]]
+            The original net merit data, $USD.
+
+        Returns
+        -------
+        dict[str, dict[str, dict[str, float]]]
+            The net merit data after filling the gap in between entries, $USD.
+        """
+        expanded_net_merit: dict[str, dict[str, dict[str, float]]] = {}
+        monthly_increase_lookup = {2005: 132 / 60, 2010: 184 / 60, 2015: 231 / 60, 2020: (360.731239 - 36.931108) / 48}
+
+        for breed in original_net_merit.keys():
+            expanded_net_merit[breed] = {}
+            years = [int(year_month[:4]) for year_month in original_net_merit[breed].keys()]
+            max_year = max(years)
+            current_keys = list(original_net_merit[breed].keys())
+            for year_month in current_keys:
+                expanded_net_merit[breed][year_month] = {
+                    "average": original_net_merit[breed][year_month]["average"],
+                    "std": original_net_merit[breed][year_month]["std"],
+                }
+                year, month = int(year_month[:4]), int(year_month[5:])
+
+                if month < 12:
+                    month += 1
+                else:
+                    year += 1
+                    month = 1
+                next_year_month = str(year) + "-" + str(month).zfill(2)
+                num_inc = 1
+                while next_year_month not in current_keys:
+                    average_monthly_increase_key = year - (year % 5)
+                    average_monthly_increase = monthly_increase_lookup[average_monthly_increase_key]
+                    expanded_net_merit[breed][next_year_month] = {
+                        "average": original_net_merit[breed][year_month]["average"]
+                        + num_inc * average_monthly_increase,
+                        "std": original_net_merit[breed][year_month]["std"],
+                    }
+                    if month < 12:
+                        month += 1
+                    else:
+                        year += 1
+                        month = 1
+                    next_year_month = str(year) + "-" + str(month).zfill(2)
+                    num_inc += 1
+                    if year > max_year:
+                        break
+
+            updated_keys = list(expanded_net_merit[breed].keys())
+            updated_keys.sort()
+            expanded_net_merit[breed] = {k: expanded_net_merit[breed][k] for k in updated_keys}
+        return expanded_net_merit
+
+    @staticmethod
+    def assign_net_merit_value_to_animals_entering_herd(birth_date: str, breed: str) -> float:
+        """
+        This function calculates the net merit value for animals entering the herd, either during initialization or
+        for animals bought during the simulation.
+
+        Parameters
+        ----------
+        birth_date: str
+            The birthdate of the animal in the format "%Y-%m-%d".
```

Comment:
```suggestion
            The birthdate of the animal in the format "YYYY-MM-DD".
```
If this is the full format of the dates I think it would be better to specify it fully.

Reply:
Edit: after seeing how `strftime` is used I understand why it is written as "%Y-%m-%d" here. But because this is documentation and not code, I think using the above would be better because it is more readable and consistent with how date formats are documented elsewhere.